graders/c/cgrader/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+
+# This script is the entrypoint for the grader. It assumes most users will
+# include a /grade/tests/test.py file as a starting point, and will invoke that
+# file if it exists. 
+
+if [[ -r /grade/tests/test.py ]]; then
+    python3 /grade/tests/test.py
+    exit $?
+fi
+
+# If /grade/tests/test.py does not exist, but there is a single .py file in the
+# /grade/tests directory, run that file. 
+
+single_py_file=$(find /grade/tests -maxdepth 1 -name '*.py' -type f -readable)
+if [[ $(echo "$single_py_file" | wc -w) -eq 1 ]]; then
```

Comment:
Shouldn't you use

```
-print -quit
```
on the find, and then https://unix.stackexchange.com/questions/62880/how-to-stop-the-find-command-after-first-match
```
if [ -f $single_py_file ]; then
```

Reply:
No, I don't want to find the first, I want to check if there's only one. It should error out if there's more than one.